Neos.Neos/Classes/Domain/Service/UserService.php

```diff
@@ -59,6 +60,12 @@ class UserService
      */
     protected $workspaceRepository;
 
+    /**
+     * @Flow\Inject
+     * @var PersistenceManagerInterface
+     */
+    protected $persistenceManager;
```

Comment:
Would not be required if we can obtain the current user id from the `UserService`